packages/accounts-password/password_server.js

```diff
@@ -67,6 +67,15 @@ Accounts._checkPassword = function (user, password) {
 
   if (! bcryptCompare(password, user.services.password.bcrypt)) {
     result.error = handleError("Incorrect password", false);
+  } else {
```

Comment:
Since there is only one `if` under `else` here, could you merge them into an `else if` to improve readability a bit?

Reply:
Makes sense - done in a5206a6